services/payments/functions/process-transaction/main.py

```diff
@@ -0,0 +1,108 @@
+import boto3
+import json
+import base64
+from urllib.parse import parse_qs
+import traceback
+
+AWS_REGION = "us-east-1"
+dynamoClient = boto3.client('dynamodb', region_name=AWS_REGION)
+
+
+"""
+Request is the class used for handling the request
+"""
+
+
+class Request:
+
+    def __init__(self):
+        self.err = None
+
+    def isValidateTransaction(self):
+        if self.message is None:
+            return False
+
+        user_id = self.message.get("user_id", "")
+        if user_id == "":
+            return False
+
+        user_name = self.message.get("user_name", "")
+        if user_name == "":
+            print("not user name")
+            return False
+
+        reference_id = self.message.get("reference_id", "")
+        if reference_id == "":
+            return False
+
+        description = self.message.get("description", "")
+        if description == "":
+            return False
+
```

Comment:
I think you can put them in a dictionary in order to leave out many conditions

Reply:
done